common/doc.go

```diff
@@ -1,3 +1,29 @@
 // Package common contains wrapper types and utilities common to all of the
 // OOXML document formats.
 package common
+
+import (
+	"archive/zip"
+	"strings"
+
+	"github.com/mec07/unioffice/zippkg"
+	"github.com/unidoc/unioffice"
+)
+
+//AddImageToZip adds an image (either from bytes or from disk) and adds it to the zip file.
+func AddImageToZip(z *zip.Writer, img ImageRef, imageNum int, dt unioffice.DocType) error {
+	fn := unioffice.AbsoluteFilename(dt, unioffice.ImageType, imageNum)
+	fn = fn[0:len(fn)-3] + strings.ToLower(img.Format())
```

Comment:
I'm not a massive fan of this line. It is here to replace the `png` with the new image format. The `png` is set here: https://github.com/unidoc/unioffice/blob/master/filenames.go#L148-L155...

Reply:
I've fixed this by adding `AbsoluteImageFilename`. I don't know if it is the best solution, but it works. The other option is to add an additional parameter to the `AbsoluteFilename` function, which is used all over the place (and it's only for images that the additional parameter would be required).